docs/how-to/h-enable-encryption.md

```diff
@@ -16,9 +17,9 @@ juju deploy self-signed-certificates --channel=edge
 juju config self-signed-certificates ca-common-name="Test CA"
 ```
 
-Please refer to [this post](https://charmhub.io/topics/security-with-x-509-certificates) for an overview of the TLS certificates Providers charms and some guidance on how to choose the right charm for your use-case. 
+Please refer to [this post](https://charmhub.io/topics/security-with-x-509-certificates) for an overview of the TLS certificates Providers charms and some guidance on how to choose the right charm for your use case. 
 
-## Enable TLS on Kafka and ZooKeeper
+## Relate the charms
```

Comment:
**todo:** I would revert this change. I want to know your thoughts on this though, but to me, it's more useful to title a section with what you're achieving, not the actions outlined in the section itself.

Reply:
There are two main reasons for this change.
This is the How to enable encryption guide for Charmed Apache Kafka guide. So an h2 heading of `Enable TLS on Charmed Apache Kafka and Charmed Apache ZooKeeper` 
(a) seems a bit too generic and repetitive - I'd rather not have a step `Enable encryption` in a `How to enable encryption` guide. Why not announce the action needed in the heading? It's the best name for a step in a step-by-step instruction.
(b) seems way too long - I'd like to keep headings 4-5 words at max if possible.
Do you have an alternative idea for it?

Reply:
Nah, it's fine to keep as is, your reasoning makes sense 👍🏾